vj4/model/record.py

```diff
@@ -75,6 +75,16 @@
 
 
 @argmethod.wrap
+async def set_visibility(rid: objectid.ObjectId, visibility: int):
+  if visibility not in constant.setting.SUBMISSION_VISIBILITY_RANGE:
+    raise error.ValidationError('visibility')
+  coll = db.coll('record')
+  doc = await coll.find_one_and_update(filter={'_id': rid},
```

Comment:
doc is not used, please either remove the variable definition, or return something